.github/workflows/push.yml

```
@@ -29,14 +29,14 @@ jobs:
         uses: docker://php:7.3-alpine
         with:
           entrypoint: vendor/bin/phpstan
-          args: analyse
+          args: analyse --memory-limit 1G
```

```suggestion
          args: analyse --memory-limit=-1
```

Can we [configure PHPStan](https://phpstan.org/user-guide/command-line-usage#--memory-limit) to [configure PHP not to have a memory limit](https://www.php.net/manual/en/ini.core.php#ini.sect.resource-limits)?

---

We can try :)